DatadogSDKBridge/Classes/Implementation/DdRumImplementation.swift

```diff
@@ -137,4 +190,15 @@ internal class DdRumImplementation: DdRum {
         context[Self.timestampKey] = timestampMs
         return castAttributesToSwift(context)
     }
+
+    private func timingValue(from timings: [String: Any], for timingName: String) -> (start: Date, end: Date)? {
+        let timing = timings[timingName] as? [String: NSNumber]
+        if let startInNs = timing?["startType"]?.int64Value, let durationInNs = timing?["duration"]?.int64Value {
+            return (
+                Date(timeIntervalSince1970: TimeInterval(Double(startInNs) / 1_000_000_000)),
```

Comment:
Don't we want the nanoseconds precision here or `TimeInterval` accepts only milliseconds ?

Reply:
it accepts only seconds :) but there is no loose of the precision, because it takes seconds in the double format.

Reply:
Oh...I see ;)